### PR TITLE
Add support for detected boxes

### PR DESCRIPTION
**Pull request by kakul**
Fix click behavior

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -27,3 +30,21 @@ export const imageDataToDataUrl = (imageData: ImageData): string | null => {
 
   return canvas.toDataURL();
 };
+
+export const getAbsoluteDetectedBoxes = (
+  parent: CropperBox,
+  children: Array<Array<number>>
```
Comment (jayantbh) [inline src/utils.ts:36]:
```suggestion
  children: Array<[number, number, number, number]>
```

File: src/components/MultiCrops.tsx [hunk 2]
```diff
@@ -329,10 +329,16 @@ const MultiCrops: FC<CropperProps> = ({
...
-      e.target === imageRef.current || e.target === containerRef.current;
-    if (!isTargetInCropper) return;
-
+    if (
+      e.nativeEvent
+        .composedPath()
+        .findIndex((elem) => elem === imageRef.current) === -1 &&
+      e.nativeEvent
+        .composedPath()
+        .findIndex((elem) => elem === containerRef.current) === -1
```
Comment (jayantbh) [inline src/components/MultiCrops.tsx:338]:
```suggestion
      !e.nativeEvent.composedPath().includes(imageRef.current) &&
      !e.nativeEvent.composedPath().includes(containerRef.current)
```

File: src/components/MultiCrops.tsx [hunk 3]
```diff
@@ -330,12 +330,10 @@ const MultiCrops: FC<CropperProps> = ({
...
-      e.nativeEvent
-        .composedPath()
-        .findIndex((elem) => elem === imageRef.current) === -1 &&
-      e.nativeEvent
-        .composedPath()
-        .findIndex((elem) => elem === containerRef.current) === -1
+      imageRef.current &&
+      !e.nativeEvent.composedPath().includes(imageRef.current) &&
+      containerRef.current &&
+      !e.nativeEvent.composedPath().includes(containerRef.current)
```
Comment (kakul) [inline src/components/MultiCrops.tsx:336]: Added null check to the suggestion.

**Comment by kakul** [comment]
<table>
  <tr><td></td></tr>
  <tr><td><h1 align="center">ARIGATOU!</h1></td></tr>
<table>

**Pull request merged**